src/renderer/pages/camera_setup/index.tsx

```diff
@@ -0,0 +1,129 @@
+import PTZOpticsPower from '../../../../assets/photos/ptzoptics_power.jpg';
+
+const subSteps = [
+    { text: "Time to setup the camera(s)!  Locate the orange camera pelican and take out a camera.", img: CameraPelicans },
+    { text: "For FRC, the camera should be placed at least 15 feet from the field border. For FTC, the cameras should be roughly 8 feet from the field.", img: FRCField },
+    { text: "Survey your location and find a good spot for the camera. The camera can be placed either on one of the provided tripods, or using one of the pole mounts.  Note: you only have about 100 feet of ethernet, so keep that in mind!", img: FRCField },
+    { text: "Once placed, run an ethernet cable to each of the cameras (if you are using multiple) and plug them to a highlighted port (port 13 through 22) of the purple networking switch on the backside of the AV Cart.", img: AVVlanHighlighted },
+    { text: "Once plugged in, the cameras should spin and come online.  If the camera does not spin, and does not come online below, check that the power switch on the back is flipped on.", img: PTZOpticsPower },
+    { text: "If you are still having issues, please contact support for further assistance.  You will be able to continue once a camera has been detected below.", img: null }
+]
+
+function InternetSetup({ nextStep, previousStep }: Steppable) {
+    const [errors, setErrors] = useState<string[]>([]);
+    const [cam1, setCam1] = useState<boolean>(false);
+    const [cam2, setCam2] = useState<boolean>(false);
+    const [subStep, setSubStep] = useState<number>(0);
+
+    useEffect(() => {
+        // Fire off HW check and register listener
+        window.electron.ipcRenderer.on(
+            'hw-change',
+            (arg: HWPingResponse) => {
+                setErrors(arg.errors);
+                setCam1(arg.camera1);
+                setCam2(arg.camera2);
+            }
+        );
+        window.electron.ipcRenderer.sendMessage('hw-status', []);
+    }, []); // only run once
+
+    const previous = () => {
+        if (subStep > 0) {
+            setSubStep(subStep - 1);
+        }
+    }
+
+    const next = () => {
+        if (subStep < subSteps.length - 1) {
+            setSubStep(subStep + 1);
+        }
+    }
+
+    return (
+        <>
+            <Space
+                direction="vertical"
+                style={{ textAlign: 'center', width: '100%' }}
+            >
+                <Typography.Title level={1}>
+                    Camera Setup
+                </Typography.Title>
+
+                <Flex gap="small" align="center" vertical>
+                    <Flex gap="small" wrap="wrap">
+                        <Button onClick={previous} disabled={subStep === 0}>
+                            Previous
+                        </Button>
+                        {subStep < subSteps.length - 1 && (
+                            <Button onClick={next} disabled={subStep === subSteps.length - 1}>
```

Comment:
Seems like disabled would never be true here?

```suggestion
                            <Button onClick={next}>
```